discoverable-client/src/test/java/com/ca/mfaas/client/api/PetControllerGetOneTest.java

```diff
@@ -24,9 +24,7 @@
 import static org.hamcrest.Matchers.hasItem;
 import static org.hamcrest.core.Is.is;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
It's better not import every static method from `org.mockito.Mockito`, because there are too many, but we need only few of them.

Reply:
Fixed this one